dallinger/experiment_server/experiment_server.py

```diff
@@ -573,9 +573,9 @@ def create_participant(worker_id, hit_id, assignment_id, mode):
         app.logger.warning(msg.format(duplicate.id))
         q.enqueue(worker_function, "AssignmentReassigned", None, duplicate.id)
 
-    # Count participants waiting for quorum
+    # Count working participants
```

Comment:
@alecpm I think `create_participants()` has been pretty heavily refactored on the development branch, so whatever merge or rebase happens to incorporate these changes there might need some extra TLC.

Reply:
Fun!